packages/components/src/components/editor/condition-builder/Condition.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button } from '@/components/common/button/button';
+import { BasicField } from '@/components/common/field/field';
+import { Flex } from '@/components/common/flex/flex';
+import { BasicSelect } from '@/components/common/select/select';
+import {
+  useConditionBuilderContext,
+  type ConditionType,
+  type LogicalOperator
+} from '@/components/editor/condition-builder/ConditionBuilderContext';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { Label } from '@radix-ui/react-label';
+
+export interface Condition {
```

Comment:
I think it is not idea that the type here has the same name as the Component.

Reply:
i renamed the Condition interface and the ConditionGroup interface to "ConditionData" and "ConditionGroupData". What do you think?